app/src/main/java/com/eveningoutpost/dexdrip/StartNewSensor.java

```diff
@@ -54,6 +54,7 @@ public class StartNewSensor extends ActivityWithMenu {
     // private TimePicker tp;
     final Activity activity = this;
     Calendar ucalendar = Calendar.getInstance();
+    private int transmitterAgeInDays = DexTimeKeeper.getTransmitterAgeInDays(getTransmitterID());
```

Comment:
I suggest this should be a private method returning the value so that it can be used in both places it is needed as it can be updated dynamically in the background after the class has been instantiated but before the user clicks anything.

Reply:
Should the private method be inside Home.java?
Or, should it be in StartNewSensor.java?

Reply:
private method inside this class. The use of method is just to avoid duplication but also to allow dynamic results as it can change in the background. 

```
private int getTransmitterAgeInDays() {
   return DexTimeKeeper.getTransmitterAgeInDays(getTransmitterID());
```

Then replace variable use by call to that method

Reply:
Thanks - Done